### PR TITLE
[bitnami/sealed-secrets] add sealed secrets chart

### PR DESCRIPTION
**Pull request by onedr0p**
**Description of the change**

Adds sealed-secrets helm chart, copied from https://github.com/helm/charts/tree/master/stable/sealed-secrets

**Benefits**

Seeing as the official Helm chart repo will be deprecated on Nov 13th 2020 is makes sense to migrate here.

Closes https://github.com/bitnami/charts/issues/4016
Closes https://github.com/bitnami-labs/sealed-secrets/issues/389

**Possible drawbacks**

None

**Applicable issues**

None

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Comment by rafariossaa** [comment]
Hi,
  Thank you very much for you PR,
  I have seen you opened this issue: https://github.com/bitnami-labs/sealed-secrets/issues/389, and it is not solved yet.
  Did you have any luck reaching out the current maintainer ?

  I would like to have he/she approval before taking any further actions.

**Comment by onedr0p** [comment]
Thanks, I'll reach out via email and get their thoughts.

**Comment by onedr0p** [comment]
I sent an email to @olib963 and @stefanprodan

**Review by stefanprodan — APPROVED** [review]
LGTM

I think this repo is the perfect place for sealed-secrets 💯

**Review by olib963 — APPROVED** [review]
Seems like a logical move :smile:

**Comment by mkmik** [comment]
Ok, for technical reasons it cannot belong to this catalog, we'll move it directly into the sealed-secrets repository.

@onedr0p could you please create a PR against https://github.com/bitnami-labs/sealed-secrets and put the helm chart sources in a top level dir called `helm` ?

**Comment by onedr0p** [comment]
@mkmik I can do that but can you elaborate? 

It is my understanding that the https://github.com/bitnami-labs/sealed-secrets doesn't have a way to package the helm chart, or do you/team plan to add that there?

**Comment by mkmik** [comment]
yes, currently it doesn't have the machinery to publish the helm charts, but I've discussed with my colleagues in the bitnami catalogue group and for now the best course of action is for us to just add the required scripts in the release process of the official sealed-secrets repo; among other things this also means that the helm chart is actually tested using the same integration tests as the official manifests, reducing the chances of accidental config drift.

**Comment by tricky42** [comment]
I was such thinking about also doing a PR but remembered to search PRs before... Great to see that the sealed-secrets chart will find a new home. 

@mkmik so far I kind of thought of the Bitnami  Chart repo as a common home of Helm-Charts as long as a given project is not providing their own Chart repo. From a user perspective I kind of like having less places to look for a chart. As you mentioned technical reasons. can you provide more details as technically it wouldnt be a problem to add sealed secrets here, or? It is more that the project maintainers would be able to tie it closer into the general release process, or?

I just want to understand the positioning of the Bitnami Helm Chart repo better.

**Comment by mkmik** [comment]
Yes the bitnami repo is indeed the place where the Bitnami team provides helm chart for software that is not available upstream. The Bitnami team does a lot of work for each of the supported charts. They build the images, keep them up to date, configure the software, write automated tests, craft documentation and provide community support.

The sealed-secret project already comes with docker images (multi-arch), yaml manifests for easy installation with kubectl, automated integration tests for all supported k8s versions and OS platforms.

As the maintainer, I was always in two minds about the fact that there was a community maintained helm chart.

On one hand, I was happy that somebody else took care of that popular packaging format, relieving me from the burden of fiddling with a brittle textual templating engine for templating a whitespace-sensitive configuration format such as yaml (among other helm pain points, I don't want to digress; let's say I'm not what you'd call a helm enthusiast).

On the other hand, this meant that sealed secrets has two sets of manifests: the official ones and the helm chart. And they can go out of sync. It happened frequently enough to be more than a mere annoyance. Now, every time somebody reports a bug or asks for help I have to ask "official yaml or helm?"; And then try to reproduce the issue with helm. This is distracting and time consuming.

And it's entirely my fault. I let this "fork" happen. The community helm chart maintainers cannot be blamed for not keeping the templates in perfect sync; it's hard.

Moving the maintenance of the helm chart to another set of external maintainers such as Bitnami wouldn't fix the underlying problem.

So, I'm taking this opportunity to move the helm chart release process in-sync with the plain "official" yaml manifests. Both manifests will be tested with the same integration tests and will be cut at the same time by the same release manager.

I have strong commitments to backward compatibility and seamless upgrades, and when it cannot be met I want to be able to clearly indicate the conditions and required actions in the release notes. For that to work well, IMO, there has to be one and only one place to reason about the release notes.

My Bitnami colleagues will help me maintaining the charts, with their expertise and tooling, as needed, but the home of the chart belongs inside to the sealed-secrets project repo, in order to guarantee that such an infrastructure component can be relied upon and can continue to have the level of support that the community is accustomed to have.

**Pull request closed without merging**
